src/Azure/Azure.Quantum.Client/Utility/Constants.cs

```diff
@@ -0,0 +1,35 @@
+﻿// <copyright file="Constants.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+
+namespace Microsoft.Azure.Quantum.Utility
+{
+    internal static class Constants
+    {
+        internal const string DefaultBaseUri = "https://app-jobscheduler-prod.azurewebsites.net/";
+
+        internal static class Aad
+        {
+            internal const string ApplicationId = "84ba0947-6c53-4dd2-9ca9-b3694761521b";
+
+            // TODO: Confirm audience
```

Comment:
Was this confirmed?

Reply:
I'm not sure what it's referring to. @rmshaffer @anpaz-msft @cesarzc @swernli ?

Reply:
I'm not sure how old the comment is, but the `Audience` string here looks correct to me.

FWIW, all of this code under Azure.Quantum.Client has already been in master of the private repo for some time. A detailed review here shouldn't be necessary, since it should have already been reviewed when it was first written and committed.

Reply:
(The exception is `Azure.Quantum.Client/Machine/QuantumMachineFactory.cs`, which did not come from the private repo.)

Reply:
All of the code here has already been reviewed since this is just the merge of the feature branch. I am also just skimming, but even so, merging is a good opportunity to look through the code to exactly spot things here and there that god forgotten (like the todo or the licenses).

Reply:
TODO has been removed.